MantidPlot/src/Plot3DDialog.cpp

```diff
@@ -783,7 +783,7 @@ bool Plot3DDialog::updatePlot()
 
 		try {
 			MyParser parser;
-			parser.SetExpr(to.ascii());
+			parser.SetExpr(to.latin1());
```

Comment:
The reason is that I started using `latin1` before realizing the right option is `.toAscii().constData()`. I replaced all the occurrences but apparently I missed some... Thanks!